tests/js/client/communication/test-communication.js

```diff
@@ -732,6 +710,7 @@ function GenericAqlSetupPathSuite(type) {
 
   const testSuite = {
     setUpAll: function () {
+      reconnectRetry(primaryEndpoint, "_system", "root", "");
```

Comment:
if the helper methods do this reconnectRetry internally this Is not required at all